Ignore-Files-Committed-To-Git-Repo.md

```diff
@@ -0,0 +1,20 @@
+# Untrack files previously committed from new .gitignore
+To untrack a _single_ file, ie stop tracking the file but not delete it from the system use:
+
+`git rm --cached filename`
+
+To untrack _every_ file in `.gitignore`:
+
+First **commit** any outstanding code changes, and then run:
+
+`git rm -r --cached`
+
+This removes any changed files from the index(staging area), then run:
+
+`git add .`
+
+Commit it:
+
+`git commit -m ".gitignore is now working"`
+
+To undo `git rm --cached filename`, use `git add filename`
```

Comment:
Well not so much, if you removed the wrong file by accident it tells you how to add it back. The whole file could be improved content wise but the purpose of this PR was to remove the css so it could be merged. If you want, you can add an issue to improve on this one and someone can then work on it.